synspec_examples.ipynb

```diff
@@ -0,0 +1,722 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "observation_preccd = Observation(photlam_hatp11, uniform_bp, binset=spec_wavelengths)\n",
+    "observation_preccd.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot the binned photometry (aka spectroscopy) on top of the source spectrum for comparison:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 169,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<matplotlib.legend.Legend at 0x125958748>"
+      ]
+     },
+     "execution_count": 169,
+     "metadata": {},
+     "output_type": "execute_result"
+    },
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAZgAAAEKCAYAAAAvlUMdAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBo\ndHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAIABJREFUeJzsnXd4VFXawH9nZtJJI4SWEEKVDlIV\nUUEFWRsWFFz7quyq6NpFP/tacO0urmV37RU7CoqufRUVVKpIb6GGkEJ6ZuZ8f0zJnX4nmTslOb/n\n4eGWc+85mblz3vuW875CSolCoVAoFJHGFOsBKBQKhaJtogSMQqFQKAxBCRiFQqFQGIISMAqFQqEw\nBCVgFAqFQmEISsAoFAqFwhCUgFEoFAqFISgBo1AoFApDUAJGoVAoFIZgifUAIkWnTp1kcXFxrIeh\nUCgUCcXPP/+8X0qZb8S924yAKS4uZtmyZbEehkKhUCQUQohtRt1bmcgUCoVCYQhKwCgUCoXCEJSA\nUSgUCoUhKAGjUCgUCkNQAkahUCgUhqAEjEKhUCgMQQkYhUKhUBhCm1kHo1C0FCklv+6oYOmWA7y0\nZBt2KTl9ZAFPfrkJgNE9c9laVsP+6saw7puXkcxT545iWGE2VrskPcmMySSM+BMUirhESCljPYaI\nMHr0aKkWWioC0Wi1s7WshimPfuNxvFOHFPZXN8RoVKE5eXh3Hpw+jBSLCSGUcFJEHiHEz1LK0Ubc\nW2kwijbDgZpG3vm5hHsXrdV9TTwLF4APV+ziwxW7Ap6f98dDOax3Hp06pERxVAqFPpSAUSQcUkrq\nmmxMeugr9lZFVkAMTttIrvkg/6seAQhOzP6WNXW92dpYoOt6EzZSRBN1MjWi4wrE7Nd+DXr+qP75\n/GPmoWSnJ0VlPAqFFmUiU8Q1Ukr63/oxTbbIPqdpot4tBJJFE3d2f4a3yo/jvb7XA/DgnvP4V+np\nrB96GnX2FAaufkc7Kuf/viarewvmcU7eJ5y+8UG2NHSn3JYdgdFKru7yGu+WH8P2xm4ATMxcSs/k\nPXxadRi7mzoB0NFcxQFblt9x+eOZ80Zx/OCuERifIpEx0kSmBIwi7vhy3T4uen6pYfcflraeBf2u\n5YLNd7GrqRN/L3yCQzPWebR
```

Comment:
Binned flux should be visualized as binned.
```suggestion
    "plt.step(observation_preccd.binset, observation_preccd.binflux, where='mid', color='orange', label='observed spectrum')\n",
```

See https://matplotlib.org/3.1.0/gallery/lines_bars_and_markers/step_demo.html

Reply:
Ah thanks, I forgot how to plot steps! (turns out the bins are too small to make a noticeable difference in this example but it'll be good for reference anyway)

Reply:
Is there value in plotting a second plot that is zoomed into some feature of interest (H-alpha or something popular) to show the steps?

Reply:
Oh yeah that's a good idea - I think that's valuable.